tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -362,19 +367,23 @@ def __validate_case_closure_employment(self, t4, t5s, error_msg):
                 t4_record,
                 "EMPLOYMENT_STATUS",
                 line_num,
-                error_msg
+                error_msg,
+                deprecated=True
```

Comment:
confirmed the following no longer appears on the error report ✔️

`At least one person on the case must have Item 28 (Employment Status) = 1:Yes in the same Item 4 (Reporting Year and Month) since Item 9 (Reason for Closure) = 1:Employment/excess earnings.`